File: 1816.truncate-sentence.py
#
# @lc app=leetcode id=1816 lang=python
#
# [1816] Truncate Sentence
#

# @lc code=start
class Solution(object):
    def truncateSentence(self, s, k):
        """
        :type s: str
        :type k: int
        :rtype: str
        """
        r = ''
        c = 0
        for i in s:
            if i == ' ':
                c+=1
            if c == k:
                break
            r += i
        return r
        
# @lc code=end

File: 1816.truncate-sentence.java
/*
 * @lc app=leetcode id=1816 lang=java
 *
 * [1816] Truncate Sentence
 */

// @lc code=start
class Solution {
    public String truncateSentence(String s, int k) {
        StringBuilder sb = new StringBuilder();
        int co = 0;
        for (char c : s.toCharArray()) {
            if (c == ' ') {
                co++;
            }
            if (co == k) {
                break;
            }
            sb.append(c);
        }
        return sb.toString();
    }
}
// @lc code=end

